.coderabbit.yaml

```
@@ -0,0 +1,31 @@
+
```

YAMLlint (1.37.1)

[error] too many blank lines (1 > 0)

(empty-lines)